ci/jenkins/Jenkinsfile_unix_gpu

```diff
@@ -53,6 +53,7 @@ core_logic: {
     custom_steps.test_unix_python3_tensorrt_gpu(),
     custom_steps.test_unix_perl_gpu(),
     custom_steps.test_unix_r_gpu(),
+    custom_steps.test_unix_r_mkldnn_gpu(),
```

Comment:
I'm not sure whether we need GPU+MKLDNN for R since we already test it in python. Could you elaborate what concerns you are specifically having?